flagsmith-jira-app/src/IssueFlagPanel.tsx

```diff
@@ -27,8 +27,8 @@ import {
   EnvironmentModel,
   FLAGSMITH_APP,
   FeatureModel,
-  FeatureStateValue,
-  FlagModel,
+  // FeatureStateValue,
+  // FlagModel,
```

Comment:
These should be removed if they're not necessary.

Reply:
This was already changed in the latest version